tools/wpt/browser.py

```diff
@@ -239,11 +252,14 @@ def platform_string(self):
 
         return "%s%s" % (platform, bits)
 
+    def find_binary(self):
+        # ChromeDriver can find browser binary itself.
```

Comment:
I take it this is supposed to be a no-op change and it probably is, but changing this kind of code does me nervous about ways in which wpt.fyi running might run. Are there any tests at all for this stuff? @gsnedders?

Reply:
An actual no-op would be `return None`. Throwing an exception would prevent future accidental use of these methods (these methods are not called currently).

I don't think there are any tests specifically for the stuff. But we do have integration tests for major products, don't we?